webapp/src/main/java/com/box/l10n/mojito/smartling/SmartlingOAuthAccessResponse.java

```diff
@@ -0,0 +1,119 @@
+package com.box.l10n.mojito.smartling;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class SmartlingOAuthAccessResponse {
```

Comment:
records everywhere :)

Reply:
I can't use records here as I need to set the expiry times on the object when I receive the response from Smartling (as their response indicates the number of seconds the token is active not the expiry time)